HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/URLOpener.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  URLOpener.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+public struct URLOpener {
```

Comment:
[Suggestion]: we need to document it since it's public :)

Reply:
Yes. Maybe we should. We need this to be public because it's used in the public method called `openPaymentProviderApp`